main.tf

```diff
@@ -0,0 +1,192 @@
+module "label" {
+  source     = "github.com/cloudposse/terraform-terraform-label.git?ref=master"
+  attributes = "${var.attributes}"
+  delimiter  = "${var.delimiter}"
+  name       = "${var.name}"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  tags       = "${var.tags}"
+}
+
+# ECR repository
+resource "aws_ecr_repository" "app" {
+  name = "${module.label.id}"
+}
+
+# Cloudwatch Log Group
+resource "aws_cloudwatch_log_group" "app" {
+  name = "${module.label.id}"
+
+  tags {
+    Stage       = "${module.label.stage}"
+    Application = "${module.label.name}"
+  }
+}
+
+# ECS Task def
+data "template_file" "default_task" {
+  template = "${file("task_definition.json")}"
+
+  vars {
+    image     = "${aws_ecr_repository.app.repository_url}"
+    log_group = "${aws_cloudwatch_log_group.app.name}"
+  }
+}
+
+resource "aws_ecs_task_definition" "default" {
+  family                = "${var.family}"
+  container_definitions = "${data.template_file.default_task.rendered}"
+
+  #container_definitions = "${module.container_definition.container_definitions}"
+  requires_compatibilities = ["${var.launch_type}"]
+  network_mode             = "${var.network_mode}"
+  cpu                      = "${var.task_cpu}"
+  memory                   = "${var.task_memory}"
+  execution_role_arn       = "${aws_iam_role.ecs_execution_role.arn}"
+  task_role_arn            = "${aws_iam_role.ecs_execution_role.arn}"
+}
+
+# ALB
+
+## Target Group
+resource "random_id" "target_group_suffix" {
+  byte_length = 2
+}
+
+resource "aws_alb_target_group" "alb_target_group" {
+  name        = "${module.label.stage}-alb-target-group-${random_id.target_group_suffix.hex}"
```

Comment:
`alb-target-group` is actually implied by the resource type, so it is redundant to have in the name. Just drop it.